application/modules/clients/controllers/Clients.php

```diff
@@ -253,4 +267,12 @@ public function delete($client_id)
         $this->mdl_clients->delete($client_id);
         redirect('clients');
     }
+
+    private function get_client_title_choices(): array
+    {
+        return array_map(
```

Comment:
@naui95 
so if we alllow fn functions, we cannot support PHP 7.4 anymore, correct?

@KevinJ30 I really like this function man, kudos!

Reply:
@nielsdrost7 `fn` functions have been introduced in PHP 7.4 so we should be ok with them. Starting from IP1.6 we also state that minimum PHP version is 8.0 so this should not be an issue.